code/modules/surgery/generic_steps.dm

```diff
@@ -2,13 +2,19 @@
 //make incision
 /datum/surgery_step/incise
 	name = "make incision"
-	implements = list(/obj/item/weapon/scalpel = 100, /obj/item/weapon/melee/energy/sword = 75, /obj/item/weapon/kitchen/knife = 65, /obj/item/weapon/shard = 45)
+	implements = list(/obj/item/weapon/scalpel = 100, /obj/item/weapon/melee/energy/sword = 75, /obj/item/weapon/kitchen/knife = 65,
+		/obj/item/weapon/shard = 45, /obj/item = 30) // 30% success with any sharp item.
 	time = 16
 
 /datum/surgery_step/incise/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
-	user.visible_message("[user] begins to make an incision in [target]'s [parse_zone(target_zone)].", "<span class='notice'>You begin to make an incision in [target]'s [parse_zone(target_zone)]...</span>")
+	user.visible_message("[user] begins to make an incision in [target]'s [parse_zone(target_zone)].",
+		"<span class='notice'>You begin to make an incision in [target]'s [parse_zone(target_zone)]...</span>")

```

Comment:
@coiax it's usually accepted for visible_message, since it's two separate sentences